public/content/developers/docs/apis/javascript/index.md

```diff
@@ -80,7 +80,7 @@ Here's an examples from Ethers
 // Create a wallet instance from a mnemonic...
 mnemonic =
   "announce room limb pattern dry unit scale effort smooth jazz weasel alcohol"
-walletMnemonic = Wallet.fromMnemonic(mnemonic)
+walletMnemonic = Wallet.fromPhrase(mnemonic)
```

Comment:
Looks good! ([supporting docs here](https://docs.ethers.org/v6/api/wallet/#Mnemonic_fromPhrase) for reference)